projects/uitdatabank/reference/entry.json

```diff
@@ -2152,6 +2152,76 @@
         ]
       }
     },
+    "/events/{eventId}/organizer/{organizerId}": {
+      "parameters": [
+        {
+          "$ref": "#/components/parameters/eventId"
+        },
+        {
+          "$ref": "#/components/parameters/organizerId"
+        }
+      ],
+      "delete": {
+        "summary": "organizer - delete",
+        "description": "Deletes the organizer of the event with the given `eventId`.",
+        "operationId": "events-organizer-delete",
+        "tags": [
+          "Events"
+        ],
+        "responses": {
+          "204": {
+            "description": "No Content. The organizer has been deleted successfully."
+          },
+          "401": {
+            "$ref": "#/components/responses/Unauthorized"
+          },
+          "403": {
+            "$ref": "#/components/responses/Forbidden"
+          },
+          "404": {
+            "$ref": "#/components/responses/NotFound"
+          }
+        },
+        "security": [
+          {
+            "USER_ACCESS_TOKEN": []
+          },
+          {
+            "CLIENT_ACCESS_TOKEN": []
+          }
+        ]
+      },
+      "put": {
+        "summary": "organizer - update",
+        "description": "Updates the organizer of the event with the given `eventId`.  You can find a list of organizers using [`GET /organizers`](reference/search.json/paths/~1organizers/get).\n\n<!-- theme: info -->\n\n> An organizer is not required on an event, and it can only have one.",
```

Comment:
Maybe better to link to this new guide instead of the endpoint for searching organizers: https://docs.publiq.be/docs/uitdatabank/entry-api/organizers/finding-and-reusing-organizers

Reply:
Didn't know that existed. I agree it's better to link to that guide.